### PR TITLE
fix(icons): move svg-icons installation docs into icon overview page

### PR DESCRIPTION
**Pull request by exelarios**
## Description
This PR addresses the issue of having duplicate sources of technical documentation of icons.

## Detail
The approach is to provide a small section for the installation process at the bottom of the `Icon Overview` page, where it includes the link to the `svg-icons` repository README.

Consideration: 
1. We thought of just simply making the `Installation` in Navbar link to the Github repo's README but we find it too disruptive since it doesn't any indications the user is leaving the garden website.

2. Using the `Configuration` component but realized it doesn't necessarily convey to the user that the implementation details are within the repo's `README`.

image

## Checklist

- [ ] :ok_hand: website updates are Garden Designer approved (add the designer as a reviewer)
- [ ] :black_nib: copy updates are approved (add the content strategist as a reviewer)
- [ ] :link: considered opportunities for adding cross-reference URLs (grep for keywords)
- [ ] :wheelchair: analyzed via [axe](https://www.deque.com/axe/) and evaluated using VoiceOver
- [ ] :memo: tested in Chrome, Firefox, Safari, Edge, and IE11

### CODE REVIEW

**Review by diana-oum — APPROVED** [review]

File: src/pages/design/icons/index.mdx [hunk 1]
```diff
@@ -71,6 +71,11 @@ In certain instances where 16px icons will break the line height or will not fit
   </Do>
 </BestPractice>
 
+### Installation
+
+View the [zendeskgarden/svg-icons](https://github.com/zendeskgarden/svg-icons#garden-svg-icons---)
```
Comment (hzhu) [inline src/pages/design/icons/index.mdx:76]: Looks like there's three extra dashes at the end of the URL?

**Review by bethanybeech — APPROVED** [review]

File: src/pages/design/icons/index.mdx (hunk 1 above)
Comment (hzhu) [reply src/pages/design/icons/index.mdx:76]: Oh I see, it's from GitHub 👀

**Review by hzhu — APPROVED** [review]

File: src/pages/design/icons/index.mdx (hunk 1 above)
Comment (jzempel) [reply src/pages/design/icons/index.mdx:76]: I think https://github.com/zendeskgarden/svg-icons#readme is the preferred URL here.

Comment (exelarios) [reply src/pages/design/icons/index.mdx:76]: Haha no wonder I was like.. isn't there a better URL that points to the readme. I'll push up a PR to fix this.

**Pull request merged**